public/layouts/dashboard/dashboard-create.js

```diff
@@ -20,9 +20,16 @@ angular
         };
 
         $scope.subscriptionsSupported = subscriptionsSupported();
+        $scope.subscriptionStatus = null;
+
         $scope.registerSubscription = () => {
-            // TODO MRB: show spinner and success or similar
-            registerSubscription();
+            $scope.subscriptionStatus = "Subscribing...";
```

Comment:
minor: this could be a `bool` in the controller, allowing the view to remain in charge of what's rendered

Reply:
It could and probably should be but I'm going to leave it to avoid too much copy-pasting in the template (two `ng-if` statements on seperate `spans`)